stdlib/Compflags

```diff
@@ -27,10 +27,10 @@ case $1 in
     # never instrument camlinternalOO or camlinternalLazy (PR#7725)
   stdlib__Buffer.cmx) echo ' -inline 3';;
                            # make sure add_char is inlined (PR#5872)
-  stdlib__Buffer.cm[io]) echo ' -w A';;
-  camlinternalFormat.cm[io]) echo ' -w Ae';;
+  stdlib__Buffer.cm[io]) echo ' -w +A';;
+  camlinternalFormat.cm[io]) echo ' -w +A -w -fragile-match';;
```

Comment:
Why not the straightforward translation to `+A-e`, if only as an example of quickly fixing existing code?

Reply:
Essentially because I did not remember what was the `e` option.